_posts/2024-01-12-release-0_4_1.md

```diff
@@ -0,0 +1,35 @@
+
+This release also includes some other small changes cleaning up examples and updating dependencies. 
+
+The whole Kroxylicious team is very excited to see people testing out the proxy and is really looking forward to hearing more about how & where people want to use it.
+
+### Encryption at Rest
+
+The `EnvelopeEncryption` filter works by intercepting all produce requests from applications and encrypts the Kafka records as they pass through Kroxylicious. On the consume path, the reverse happens - the filter intercepts the fetch responses and decrypts the records before they are sent to the application.
+We have more work planned to iterate on the implementation to reach production quality. Check out our [0.5.0 milestone](https://github.com/kroxylicious/kroxylicious/milestones/0.5.0) for details.
+
+The Kafka Cluster *never* sees the plain text of your records. Filter configuration is used to specify which topic(s) should be encrypted by which key.
+
+The encryption keys themselves are stored safely in a Key Management System.  0.4.0 ships with a [HashiCorp Vault](https://www.hashicorp.com/products/vault)(tm) integration.  We will provide other selected Key Management System integrations soon, however users can leverage the plug-in architecture of Kroxylicious to supply their own integrations.
+
+The diagram below shows how the system works at a high level.
+
+![image](https://github.com/kroxylicious/kroxylicious.github.io/assets/18440250/09aeb1f4-f420-4d89-b9f7-394473dddc05){:width="100%"}
+
+If this use-case interests you, here's a [short demo](https://asciinema.org/a/627637) that shows the feature in action. To try it out yourself there's some [interim instructions](https://github.com/kroxylicious/kroxylicious/blob/1e58fc703e45c7fdb0990adb583cf4aa857047cd/kubernetes-examples/envelope-encryption/README.md) to use the feature in development.  There will be more information landing on the website soon, but meanwhile, if you have questions or comments please head over to the [Slack channel](https://kroxylicious.slack.com/).
+
+{% include asciicinema.html id="627637" %}
+
+### Binary artefacts
```

Comment:
I think we should stick to the American spelling (artifact). It's what people are used to seeing (e.g. in `pom.xml`).

Reply:
Fixed in https://github.com/kroxylicious/kroxylicious.github.io/pull/27/commits/8b3e26f05624bb2d7607286307ba6856e3b5b9f7